hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/BackupRestoreConstants.java

```diff
@@ -116,6 +116,12 @@ public interface BackupRestoreConstants {
     "Specifies a custom backup location for Point-In-Time Recovery (PITR). "
       + "If provided, this location will be used exclusively instead of deriving the path from the system table.";
 
+  String OPTION_FORCE_RESTORE = "f";
+  String LONG_OPTION_FORCE_RESTORE = "force";
+  String OPTION_FORCE_RESTORE_DESC =
```

Comment:
nit: can you explain why should we support this use case if missing backup could cause data loss ?